Compatibility with PHP 7.2

quique opened the pull request:
In PHP 7.2 `Object` is a reserved word.

See
https://secure.php.net/manual/en/migration72.incompatible.php#migration72.incompatible.object-reserved-word
https://github.com/yiisoft/yii2/issues/7936

quique committed:
Compatibility with PHP 7.2

In PHP 7.2 `Object` is a reserved word.

See
https://secure.php.net/manual/en/migration72.incompatible.php#migration72.incompatible.object-reserved-word
https://github.com/yiisoft/yii2/issues/7936

asasmoyo commented:
We'll need to make sure that `BaseObject` is there in yii which we are using. Could you check the version and update `composer.json` as well?

quique committed:
Require Yii2 >= 2.0.13 in composer.json

It is not possible to allow Yii versions <2.0.13 and be compatible with PHP 7.2 or higher.

quique commented:
I updated composer.json.

I am afraid I don't understand what you mean about "checking the version".
It is not possible to allow Yii versions <2.0.13 and be compatible with PHP 7.2 or higher.

asasmoyo commented:
'the version' I was referring to was the version of Yii which support `BaseObject`. I think you're right, it starts from 2.0.13.

I think this should do, thanks for the PR!
